net/netdev/netdev_register.c

```diff
@@ -55,6 +55,7 @@
 #define NETDEV_PAN_FORMAT   "pan%d"
 #define NETDEV_WLAN_FORMAT  "wlan%d"
 #define NETDEV_WPAN_FORMAT  "wpan%d"
+#define NETDEV_MBIM_FORMAT  "wwan%d"
```

Comment:
Why is MBIN treated separately.  Shouldn't the be NETDEV_WWLAN_FORMAT?  There are other wwlan as well, right?

Reply:
I think he tried to keep the consistency, wlan could conflict case a board has WiFi and the USB GSM Modem.

Reply:
No, the corrent naming here must be  NETDEV_WWAN_FORMAT.

Reply:
Agree!